src/main/java/kr/hhplus/be/server/infrastructure/persistence/repository/SearchHistoryRepository.java

```diff
@@ -0,0 +1,32 @@
+package kr.hhplus.be.server.infrastructure.persistence.repository;
+
+import kr.hhplus.be.server.infrastructure.persistence.entity.SearchHistoryEntity;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import java.time.LocalDateTime;
+import java.util.List;
+
+
+public interface SearchHistoryRepository extends JpaRepository<SearchHistoryEntity, Long> {
+
+    @Query("SELECT s.keyword, COUNT(s) as count FROM SearchHistoryEntity s " +
+            "WHERE s.createdAt >= :fromDate " +
+            "GROUP BY s.keyword " +
+            "ORDER BY count DESC")
+    List<Object[]> findPopularKeywords(@Param("fromDate") LocalDateTime fromDate, Pageable pageable);
```

Comment:
```suggestion
    List<KeywordCountDto> findPopularKeywords(@Param("fromDate") LocalDateTime fromDate, Pageable pageable);
```
위와 같이 `Ojbect[]` 로 받게되면 타입 안정성이 떨어지게 되고 직접 캐스팅해야하는 불편함이 있습니다!